ext_tables.sql

```diff
@@ -5,7 +5,7 @@ CREATE TABLE tx_pfprojects_domain_model_project
 (
   title                varchar(255)        DEFAULT ''           NOT NULL,
   path_segment         varchar(2048)       DEFAULT ''           NOT NULL,
-  start_date           date                DEFAULT '0000-00-00' NOT NULL,
+  start_date           date                DEFAULT '1000-01-01' NOT NULL,
```

Comment:
Warum?

Reply:
MariaDB fordert ein gültiges Datum, welches mindestens der oben eingetragene Wert ist. Siehe https://mariadb.com/kb/en/date/#description

Reply:
Mit dieser Konfiguration erhalte ich im Backend beim Anlegen eines neuen Projektes die Fehlermeldung: 2: SQL error: 'Column 'start_date' cannot be null' (tx_pfprojects_domain_model_project:NEW6093bd6303fb5065750490).

Setz für start_date in der TCA mal bitte einen Defaultwert.